setup.cfg

```diff
@@ -0,0 +1,74 @@
+[metadata]
+name = bapsflib
+author = Basic Plasma Science Facility
+maintainer = Erik T. Everson
+maintainer_email = eeverson@physics.ucla.edu
+description = A toolkit for handling data collected at BaPSF
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = 3-clause BSD
+license_file = LICENSES/LICENSE.txt
+url = https://github.com/BaPSF/bapsflib
+download_url = https://pypi.org/project/bapsflib/
+project_urls =
+    BaPSF = http://plasma.physics.ucla.edu/
+    Documentation = https://bapsflib.readthedocs.io/en/latest/
+    GitHub = https://github.com/BaPSF/bapsflib
+keywords = bapsf, HDF5, lapd, physics, plasma, plasma physics, science
```

Comment:
I am unsure if I can edit, but all instances of bapsf should be changed to bmplib.

Reply:
Ignore this I see that you changed them